readme.md

```diff
@@ -0,0 +1,29 @@
+# Social media app CA
+
+[![Automated E2E Testing](https://github.com/Littlaa/social-media-client-ca/actions/workflows/e2e-test.yml/badge.svg)](https://github.com/Littlaa/social-media-client-ca/actions/workflows/e2e-test.yml)
```

Comment:
Well done on adding the badges. However, you are required to add some info about the configuration you just added. Info such how to setup your app locally, how to use the script like the test scripts etc

Reply:
Added information on configuration and scripts.